questions.md

```diff
@@ -0,0 +1,6 @@
+# Few more questions on Pull request
+
+* Is it possible to accept a pull request into a new branch?
+* Is a pull request the same as git pull?
+* How do I update a GitHub forked repository?
+* How to update a pull request from forked repo?
```

Comment:
You can update any pull request by pushing changes to the branch you want to merge into master or whatever branch you choose.